### PR TITLE
Adding msb.com to the list of online learning platforms

### PR DESCRIPTION
**Pull request by mstipetic**

### CODE REVIEW

**Comment by bradtopol** [comment]
@mstipetic Thank you for your contribution. Please sign the CLA so we can accept your PR for review.

File: content/en/docs/tutorials/online-training/overview.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ Here are some of the sites that offer online training for Kubernetes:
 
 * [Microservice Applications in Kubernetes - 100% Hands-On Experience (Linux Academy)] (https://linuxacademy.com/devops/training/course/name/learn-microservices-by-doing)
 
+* [Magic Sandbox - an interactive, hands-on Kubernetes learning platform](https://msb.com)
```
Comment (sftim) [inline content/en/docs/tutorials/online-training/overview.md]: Most of the other lines have the course name followed by company name in brackets.

Also, this line has “an interactive, hands-on Kubernetes learning platform” which seems a bit vendor pitch-y to me.

Comment (sftim) [reply content/en/docs/tutorials/online-training/overview.md]: Plus, it's sorted wrong. Magic comes before Microservice.

Comment (mstipetic) [reply content/en/docs/tutorials/online-training/overview.md]: Thanks for the comments! Happy to change the description to something that fits better, if you have any suggestions. I'm gonna sign the CLA now

**Comment by mstipetic** [comment]
@bradtopol @sftim I'm sorry this is taking a while to sort out and taking so much of your time.
I'm having trouble getting the Agreement to sign. Going through the requirements again, I see I should not have used my standard github account, but should make one with my official company domain.

@sftim As far as naming goes, it's a bit hard to specify, since we technically plan to cover the whole of Kubernetes, and are already covering a broad range of topics.

I think it was a mistake to try and make this edit through the github Web interface, and I think the cleanest way to resolve this would be to close this PR and open a new one using a proper @msb.com email, and hopefully with the Agreement signed using it. Let me know if that would be an acceptable way to proceed.

**Comment by tfogo** [comment]
Hi @mstipetic, please open a new PR if you think that's the best way forward. Please share a link to it here so we can close this one.

**Comment by Rajakavitha1** [comment]
/close

Hi @mstipetic closing this PR for now. Please feel free to open a new PR after you sign the CLA or reopen this PR after you sign the CLA.

**Pull request closed without merging**